docs/pages/router/reference/not-found.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Not Found Routes
+description: Learn how to handle server 404s and missing routes
+---
+
+> Available from Expo SDK 50 and Expo Router v3.
+
+404s can be handled by using a `+not-found.js` route. This route matches all unmatched routes from a nested level, this is akin to `[...wild].tsx`, but unlike the deep dynamic routes, `+not-found.js` is matched after API routes.
+
+On web, the server will first serve files in the following order:
+
+1. Static files in the `public/` folder.
```

Comment:
```suggestion
1. Static files in the **public** directory.
```

- Let's bold the directory name and use directory as a term for consistency as per our writing style guide.